charts/datafold/charts/dfshell/templates/deployment.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "dfshell.fullname" . }}
+  labels:
+    {{- include "dfshell.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "dfshell.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "dfshell.labels" . | nindent 8 }}
+	{{- with .Values.podLabels }}
```

Comment:
Do you see this red line here as well? Is it a warning for tab/spaces mixing?

Reply:
@cfernhout : Comes from the helm create template. Fixed it . Mix of tabs / spaces on their end.